documentation/wiki/FeatureAvailability.md

```diff
@@ -0,0 +1,24 @@
+# What is Feature Availablity?
+Feature Availability is an API that can tell you the availability status of the specific feature of the MSBuild engine. Feature names are represented by strings and availability is an enum `FeatureStatus` with the following values:
+*  `Undefined` - the availability of the feature is undefined (not in the list)
```

Comment:
When looking on this it is confusing to me what is the difference between `Undefined` and `NotAvailable`, and what list it speaks about.

How about something like:?

```
*  `Undefined` - the availability of the feature is undefined (the feature might or might not be supported by the current MSBuild engine - but the feature is unknown to the feature availability checker, so it cannot be decided).
```

and

```
`NotAvailable` - the feature is not available (unlike `Undefined`, the feature name is known to the feature availability checker and it knows the feature is not supported by current MSBuild engine)
```